go.mod

```diff
@@ -3,6 +3,9 @@ module github.com/open-telemetry/opentelemetry-collector-contrib
 go 1.18
 
 require (
+	github.com/aws/aws-sdk-go-v2 v1.11.0
+	github.com/aws/aws-sdk-go-v2/config v1.10.1
+	github.com/aws/aws-sdk-go-v2/service/s3 v1.19.0
```

Comment:
A new `go.mod` needs to be created for `github.com/open-telemetry/opentelemetry-collector-contrib/confmap/provider/s3mapprovider` in `confmap/provider/s3mapprovider`.  These should be required directly there.  There is no need to require the new module in this `go.mod` at this time as it won't be included in the contrib distribution until the collector builder gains an ability to manage config map providers.

Expect some discussion upstream on the PR regarding whether this is the appropriate import path.

Reply:
Fixed!